tree.go

```diff
@@ -1811,8 +1815,15 @@ func (n *LeafNode) serializeLeafWithUncompressedCommitments(cBytes, c1Bytes, c2B
 	// Create the serialization.
 	var result []byte
 	switch {
-	// TODO case for a group containing a single slot
-	// case count == 1:
+	case count == 1:
+		baseSize := nodeTypeSize + StemSize + 2*banderwagon.UncompressedSize + leafValueIndexSize + leafSlotSize
+		result = make([]byte, baseSize)
```

Comment:
Similar suggestion as in base branch. `baseSize` is constant, so we can avoid `make` and prob avoid an allocation by using an array.